src/views/settings/view_gui_settings.cpp

```diff
@@ -63,18 +63,24 @@ namespace big
 		{
 			if (ImGui::TreeNode("Overlay Indicators"))
 			{
+				ImGui::Separator();
 				ImGui::BeginGroup();
+				ImGui::Text("True by default");
```

Comment:
Seems unnecessary

Reply:
Up for debate obviously however I don't think this information matters after a user has modified it.